### PR TITLE
Add `georgeo` service

### PR DESCRIPTION
**Pull request by rmpato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ Take a look at [awesome-micro](https://github.com/amio/awesome-micro)! 😌
 - [f.now.sh](https://f.now.sh/) - A microservice for getting a flag icon by country code
 - [gif.now.sh](https://gif.now.sh) - A simple micro-service for Giphy's Translate API
 - [emails.now.sh](https://emails.now.sh) - Easily deploy an email delivery service to now
-- [next-time.now.sh](https://next-time.now.sh) ([src](https://github.com/billymoon/next-time)) - A very minimal zeit themed clock (demonstrates isomorphic page rendering)
+- [next-time.now.sh](https://next-time.now.sh) ([src](https://github.com/billymoon/next-time)) - A very minimal zeit themed clock (demonstrates isomorphic page rendering)  
+- [georgeo.now.sh](https://georgeo.now.sh) - A geocoding service.
```
Comment (timneutkens) [inline README.md]: Could you add the source link? https://github.com/rmpato/georgeo

**Comment by timneutkens** [comment]
This is really cool, is there a rate limit? 😄

**Comment by rmpato** [comment]
@timneutkens thanks! 
OpenStreetMap may restrict rate, intending to guarantee the service for all its users, but there is no specific rate limit defined.

Soon, I will add support for google API too, in order to provide more options. 

Thanks!

File: README.md (hunk 1 above)
Comment (rmpato) [inline README.md]: @timneutkens done :D thank you :)

**Pull request merged**